book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.book = self
+  end
```

Comment:
- Hey :wave: in our method, we need two parameters, date, and person, to keep track of the date and the person who has rented a book this establishes the belongs-to relationship. Also, when adding a book, the book itself should also be sent to the parameter to establish the has-many relationship. so our method now should be this below, kindly update yours


 ```ruby
def add_rental(person, date)
    Rental.new(date, person, self)
  end
```